backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/service/Authorizer.kt

```diff
@@ -39,4 +39,27 @@ object Authorizer {
         return (user?.role == Role.REGION_ADMIN.db_value && user.regionId == region.id) ||
             mayViewUsersInProject(user, region.projectId.value)
     }
+
+    fun mayCreateAdministrator(
+        actingAdmin: AdministratorEntity,
+        newAdminProjectId: Int,
+        newAdminRole: Role,
+        newAdminRegion: RegionEntity?
+    ): Boolean {
+        if (actingAdmin.projectId.value != newAdminProjectId) {
+            return false
+        } else if (newAdminRole == Role.NO_RIGHTS) {
+            return false
```

Comment:
Why do we have admins with NO_RIGHTS? What is their purpose?

Reply:
Currently, we don't use it. Once we allow to delete users, we would set their role to NO_RIGHTS.